README.md

```diff
@@ -42,28 +42,49 @@ a default build, you will additionally need libxml2 and libsqlite3.
 
 On Ubuntu, you can install these using:
 
-    sudo apt install -y pkg-config build-essential autoconf bison re2c \
-                        libxml2-dev libsqlite3-dev
+```shell
+sudo apt install -y pkg-config build-essential autoconf bison re2c libxml2-dev libsqlite3-dev
+```
+
+On MacOS, you can install these using:
+
+```shell
+brew install autoconf bison re2c iconv libxml2 sqlite
+```
 
 On Fedora, you can install these using:
 
-    sudo dnf install re2c bison autoconf make libtool ccache libxml2-devel sqlite-devel
+```shell
+sudo dnf install re2c bison autoconf make libtool ccache libxml2-devel sqlite-devel
+```
 
 Generate configure:
 
-    ./buildconf
+```shell
+./buildconf
+```
 
 Configure your build. `--enable-debug` is recommended for development, see
 `./configure --help` for a full list of options.
 
-    # For development
-    ./configure --enable-debug
-    # For production
-    ./configure
+```shell
+# For development
+./configure --enable-debug
+# For production
+./configure
+```
+
+Build PHP. To speed up the build, specify the maximum number of jobs using `-j` argument:
 
-Build PHP. To speed up the build, specify the maximum number of jobs using `-j`:
+On Windows:
+```shell
+make -j4
+```
 
-    make -j4
+On Unix system:
+```shell
+make -j$(nproc)
```

Comment:
`nproc` isn't also always available on every Unix; macOS doesn't include it and if you install GNU coreutils from MacPorts, that'll make it available prefixed as `nproc`. We should probably mention the other methods that David mentions or just use a simple `-j4` for everyone (and tell people to adjust.)